css/queries.css

```diff
@@ -0,0 +1,183 @@
+/* Media queries */
+@media (min-width: 768px) {
+  .all {
+    background-image: url("/image/Header-llustration-desktop.svg");
+    background-size: 100%;
+  }
+
+  .section-home {
+    padding: 0 0 20rem 0;
+    background-image: none;
+    width: 63.89%;
+    margin: auto;
+  }
+
+  .primary {
+    font-size: 3.6rem;
+    line-height: 6rem;
+    font-weight: 800;
+  }
+
+  .supporting {
+    width: 63.8%;
+    margin: auto;
+    line-height: 3.8rem;
+  }
+
+  .icons {
+    display: flex;
+    justify-content: center;
+    width: 63.8%;
+    margin: auto;
+    gap: 2rem;
+  }
+
+  .link,
+  .menu-btn {
+    visibility: hidden;
+  }
+
+  .nav-bar ul {
+    visibility: visible;
+    display: flex;
+    gap: 3.2rem;
+    list-style: none;
+  }
+
+  .nav-list {
+    font-size: 1.5rem;
+    line-height: 2rem;
+  }
+
+  .hr {
+    visibility: hidden;
+  }
+
+  .outer-grids {
+    display: grid;
+    grid-template-columns: 48% 48%;
+    gap: 2.4rem;
+    margin: 0 120px;
+  }
+
+  .flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: center;
+    gap: 8px;
+  }
+
+  .flex-container-div {
+    margin: auto;
+  }
+
+  .btn-see-project {
+    justify-content: center;
+  }
+
+  .about-me {
+    background-image: url("/image/about-me-desktop.svg");
+    background-size: cover;
+  }
+
+  .heading-about-me {
+    margin-top: 150px;
+  }
+
+  .about {
+    width: 66%;
+    margin: auto;
+    padding: 12px 0;
+  }
+
+  .my-resume {
+    width: 159px;
+    height: auto;
+    margin: auto auto 52px auto;
+  }
+
+  .language-container {
+    display: grid;
+    grid-template-columns: 48% 48%;
+    gap: 24px;
+    margin: 0 120px;
+  }
+
+  .contact-us-div {
+    display: grid;
+    grid-template-columns: 39.6% 58.5%;
+    gap: 24px;
+    margin: 0 115px;
+  }
+
+  .text-contact {
+    font-size: 30px;
+  }
+
+  .name-container {
+    display: grid;
+    grid-template-columns: auto auto;
+    gap: 20px;
+  }
+
+  .input {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .submit-btn {
+    text-align: left;
+  }
+
+  .get-in-touch {
+    display: block;
+    margin-left: 0;
+    margin-right: auto;
+  }
+
+  .email {
+    visibility: hidden;
+    height: 0;
+  }
+
+  .footer {
+    height: 56px;
+    width: 100%;
+  }
+
+  .hr-footer {
+    visibility: visible;
+  }
+
+  #firstname,
+  #lastname,
+  #email,
+  #text {
+    width: 100%;
+  }
+}
```

Comment:
- [ ] As said previously, Kindly make sure your page is a 100% responsive on standard screen size.
As you can see the way yours looks on `768px`. Consider fixing that.
`HINT: You may consider using the flexbox or grid container to place your element on the pag`
Your design :point_down: 
![Screenshot from 2022-11-10 18-43-38](https://user-images.githubusercontent.com/97100091/201157153-53d4194a-2236-4bff-a5c7-969866f32955.png)
![Screenshot from 2022-11-10 18-44-46](https://user-images.githubusercontent.com/97100091/201157246-93a46b92-2e1c-4636-bd88-860653ed09b1.png)
![Screenshot from 2022-11-10 18-44-59](https://user-images.githubusercontent.com/97100091/201157289-2953a00a-6e2a-4887-bf7b-7e4732d81ebb.png)
![Screenshot from 2022-11-10 18-46-07](https://user-images.githubusercontent.com/97100091/201157325-95d5708f-531b-46e7-b2cc-050a6847a7c3.png)

Reply:
Hi @christianbiring1, I'm not sure if you checked the same branch (may be you accidentally cloned previous branch), but my computer and my friends' computers show completely different picture. Here are the screenshots from 2 other computers. 
![IMAGE 2022-11-10 23:29:18](https://user-images.githubusercontent.com/110017001/201177286-079f52a5-1b5d-4f06-8384-c960469b84c8.jpg)

![IMAGE 2022-11-10 23:29:27](https://user-images.githubusercontent.com/110017001/201177336-b5f2ec2c-5a53-4e14-8536-42d92d438f8c.jpg)

![IMAGE 2022-11-10 23:30:19](https://user-images.githubusercontent.com/110017001/201177529-e0f52150-417d-4480-b08b-bdca64315749.jpg)

![IMAGE 2022-11-10 23:30:37](https://user-images.githubusercontent.com/110017001/201177587-f4be2ac2-71bf-408b-8d6d-d1106197b7cf.jpg)

![IMAGE 2022-11-10 23:31:11](https://user-images.githubusercontent.com/110017001/201177703-eabfcad6-c5c7-4972-b937-ff1d9ec5013d.jpg)

![IMAGE 2022-11-10 23:31:20](https://user-images.githubusercontent.com/110017001/201177730-d6ee77ca-1ced-4d39-9980-e3ac04870633.jpg)
![IMAGE 2022-11-10 23:31:27](https://user-images.githubusercontent.com/110017001/201177764-f405f777-6c6e-497f-bf83-7ac0198388fc.jpg)
![IMAGE 2022-11-10 23:31:37](https://user-images.githubusercontent.com/110017001/201177785-ed597511-b61b-42b2-8838-703daaa3a906.jpg)